R/get_cells.R

```diff
@@ -3,14 +3,18 @@ get_cells <- function(ss,
                       sheet = NULL,
                       range = NULL,
                       col_names_in_sheet = TRUE,
-                      skip = 0, n_max = Inf) {
+                      skip = 0, n_max = Inf,
+                      detail_level = c("default", "full"),
```

Comment:
This is the user-facing argument that controls how much cell data to get:

  * `"default"` gets every field that googlesheets4 consults to guess or impose cell/column type
  * `"full"` gets all fields. It's meant for someone who wants to do artisanal [unpivotr-style](https://github.com/nacnudus/unpivotr) work that involves formats (in the sense of borders, backgrounds, etc.)